_download-meta/steam-social-releases.md

```diff
@@ -0,0 +1,9 @@
+---
+---
+
+This plugin enables OpenTTD's integration with Steam. You need OpenTTD 14.0 or later.
+
+For Linux / Windows, to install the plugin, extract the download into your `social_integration` folder, under OpenTTD in your documents folder.
```

Comment:
"in your documents folder" seems overly vague

Reply:
It strongly depends on your OS.

But as always, I rather have you come up with alternatives than say: this looks wrong :D So what would you write?

Reply:
That's fair :)
I'm not entirely sure on the wording, but referencing the relevant readme section somehow?